ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Creates and init service accounts
+  block:
+    - name: Create service account
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: v1
+          kind: ServiceAccount
+          metadata:
+            name: "{{ service_account_name }}"
+
+    - name: Grant service account admin access
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: rbac.authorization.k8s.io/v1
+          kind: RoleBinding
+          metadata:
+            name: "{{ service_account_name }}"
+          roleRef:
+            kind: ClusterRole
+            name: admin
+          subjects:
+            - kind: ServiceAccount
+              name: "{{ service_account_name }}"
+  tags:
+    - init
+
+- name: Create Operator pipeline's secrets
+  no_log: yes
+  k8s:
+    state: present
+    force: yes
+    namespace: "{{ oc_namespace }}"
+    definition:
+      apiVersion: v1
+      kind: Secret
+      type: Opaque
+      metadata:
+        name: operator-pipeline-secrets-{{ suffix }}
```

Comment:
What do you think about changing this secret name to something more meaningful, eg. pyxis-auth-cert?

Reply:
I defined the name the same way as we do it with any other service. Usually, we have one openshift secret with multiple files so it is easier when mounting all secrets to DC/pod/pipeline.